recipes/matchpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "matchpy" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8de067f7537988280891bfdc8df6f498b1f314a51de3fc41c87815fa95068ce6
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - hopcroftkarp >=1.2,<2.0
+    - multiset >=2.0,<3.0
+    - setuptools_scm >=1.7.0
+    - pytest-runner
```

Comment:
Is this needed?

Reply:
Yeah, see the prior build failures. It's in the `setup_requires`.

Reply:
Okay